go/vt/topo/locks.go

```diff
@@ -383,6 +419,17 @@ func CheckShardLocked(ctx context.Context, keyspace, shard string) error {
 // lockShard will lock the shard in the topology server.
 // UnlockShard should be called if this returns no error.
 func (l *Lock) lockShard(ctx context.Context, ts *Server, keyspace, shard string) (LockDescriptor, error) {
+	return l.internalLockShard(ctx, ts, keyspace, shard, true)
+}
+
+// tryLockShard will lock the shard in the topology server.
+// It is non-blocking call for blocking call lockshard instead.
```

Comment:
Another spot where I think the description is misleading.

Reply:
If everything else looks good then I can go ahead and fix this last change. Just wanted to save CI cycles :) ..